src/components/NavigationBar/SideMenuLink.svelte

```diff
@@ -12,6 +15,7 @@
 	</div>
 {/if}
 <a
+	on:click={() => closeSideNav()}
```

Comment:
Since `closeSideNav` didn't have any argument, no need to create a new arrow function. This should work:

```
on:click={closeSideNav}
```